packages/0xcert-ethereum-proxy-contracts/src/contracts/xcert-custom.sol

```diff
@@ -36,8 +35,7 @@ contract XcertCustom is XcertToken {
     string memory _uriPostfix,
     bytes32 _schemaId,
     bytes4[] memory _capabilities,
-    address _owner,
-    address[5] memory _proxies
+    address[6] memory _addresses
```

Comment:
This variable naming is opaque. Can the separate parameters be broken out and named?

Reply:
They where in the beginning but it broke code depth complexity. Well actual first every proxy was standalone, then owner and proxies now for the best gas performance everything is in the same array.